### PR TITLE
Tutorial

### PR DESCRIPTION
**Pull request by mjsheikh**
solution_plot

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -26,8 +26,11 @@ makedocs(
         ],
         "Nonlinear Derivatives" => [
             "nonlinear_derivatives/nonlinear_diffusion.md"
-        ]
-    ]
+        ],
+        "Operator Tutorials" => [
```
Comment (ChrisRackauckas) [inline docs/make.jl:30]: Move this to right after symbolic tutorials. Tutorials before manual and API.

File: docs/make.jl [hunk 2]
```diff
@@ -26,8 +26,11 @@ makedocs(
...
+            "operator_tutorials/working_examples.md"
```
Comment (ChrisRackauckas) [inline docs/make.jl:31]: rename the file to `kdv`.

File: docs/src/operator_tutorials/working_examples.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Solving KdV Solitons
```
Comment (ChrisRackauckas) [inline docs/src/operator_tutorials/working_examples.md:1]:
```suggestion
# Solving KdV Solitons with Upwinding Operators
```

File: docs/make.jl (hunk 1 above)
Comment (mjsheikh) [reply docs/make.jl:30]: Yeah right

**Comment by mjsheikh** [comment]
solution_plot

**Review by ChrisRackauckas — APPROVED** [review]

**Pull request merged**